lib/game.rb

```diff
@@ -0,0 +1,91 @@
+require './lib/board'
+require './lib/piece'
+require './lib/turn'
+require './lib/null_piece'
+require './lib/game'
+require './lib/win_check'
+require './lib/validity'
+
+class Game
+
+  def initialize
```

Comment:
We will probably have to add configuration type arguments when we expand to multiple players and dynamic board size